libs/blocks/table/table.css

```
@@ -1,29 +1,54 @@
-:root {
-  --border-color: #D9D9D9;
+.table,
+.table.merch {
+  --border-color: #DADADA;
   --highlight-background: #F3D949;
   --hover-border-color: #357BEB;
-  --section-head-background: #E6E6E6;
-  --checkmark-color: #268E6C;
-}
+  --checkmark-color: #2C2C2C;
+  --border-radius: 16px;
 
-.table,
-.table.merch {
   max-width: 1200px;
   margin: auto;
   padding: 20px 0;
   border-color: var(--border-color);
 }
 
+.section[class *= "grid-width-"] .table {
```

Any reason for the whitespace before and after the *= 🤔
```suggestion
.section[class*="grid-width-"] .table {
```